backend/import/xml/parser.spec.js

```diff
@@ -116,12 +116,12 @@ describe("Unit tests for XML Cockatrice Parser", () => {
         IPR: {
           baseSetSize: 1,
           cards: [{
-            cmc: 4,
+            convertedManaCost: 4,
             colors: ["R", "U"],
             isAlternative: false,
             layout: "split",
             loyalty: "",
-            manaCost: "1R // 1U",
+            manaCost: "{1}{R}{1}{U}",
```

Comment:
This doesn't seem quite right?
The // means split card with two possible costs

Reply:
Keeping the split card indicator is a good idea, particularly since it'll show up in "text-only" mode. I've updated the AddManaCostBrackets function and manaSymbolRegExp to fix this; the function should now catch and preserve the split card indicator, wrapping it in spaces instead of curly brackets when it joins it to the rest of the manaCost string.